### PR TITLE
Letters API swagger docs

### PR DESCRIPTION
**Pull request by kreek**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3241

Includes a rake task to generate a swagger block from a json-schema file. Letter download endpoint is documented but its spec is skipped as apivore can't validate binary responses.

### CODE REVIEW

File: app/swagger/schemas/letter_beneficiary.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          key :required, [:attributes]
+          property :attributes, type: :object do
+            key :required, [:benefit_information, :military_service]
+            property :benefit_information, type: :object do
+              property :has_non_service_connected_pension, type: :boolean, example: true
+              property :has_service_connected_disabilities, type: :boolean, example: true
+              property :has_survivors_indemnity_compensation_award, type: :boolean, example: true
+              property :has_survivors_pension_award, type: :boolean, example: true
+              property :monthly_award_amount, type: :number, example: true
+              property :service_connected_percentage, type: :integer, example: true
```
Comment (omgitsbillryan) [inline app/swagger/schemas/letter_beneficiary.rb]: Type is `:integer` but example is `true`

File: app/swagger/schemas/letter_beneficiary.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        property :data, type: :object do
+          key :required, [:attributes]
+          property :attributes, type: :object do
+            key :required, [:benefit_information, :military_service]
+            property :benefit_information, type: :object do
+              property :has_non_service_connected_pension, type: :boolean, example: true
+              property :has_service_connected_disabilities, type: :boolean, example: true
+              property :has_survivors_indemnity_compensation_award, type: :boolean, example: true
+              property :has_survivors_pension_award, type: :boolean, example: true
+              property :monthly_award_amount, type: :number, example: true
```
Comment (omgitsbillryan) [inline app/swagger/schemas/letter_beneficiary.rb]: example and type don't match

File: app/swagger/requests/letters.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          )
+
+          parameter :authorization
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Options to include in generated PDF'
+            key :required, false
+
+            schema do
```
Comment (omgitsbillryan) [inline app/swagger/requests/letters.rb]: What is `schema`?  How does it compare to `swagger_schema`?

Comment (kreek) [inline app/swagger/requests/letters.rb]: `schema` is for inline block definition of a json object within a parameter (in this case the body), `swagger_schema` is for a model.

**Pull request merged**